docs/src/proposals.md

```diff
@@ -1 +1,55 @@
 # Proposals
+
+The core of Konoha are proposals. This is one of the two components (the other being [upgrades](upgrades.md) that should be mixed into every deployed instance.
+
+## Constants
+
+To propose a proposal, you need 1/200 of the total supply of the voting token.
```

Comment:
"To propose a proposal, you need voting power equivalent to 1/200th of the total supply of voting tokens.

By default, proposals have a one-week voting period"